keps/sig-auth/2799-reduction-of-secret-based-service-account-token/kep.yaml

```diff
@@ -13,10 +13,10 @@ reviewers:
 approvers:
   - "@liggitt"
 stage: beta
-latest-milestone: "v1.26"
+latest-milestone: "v1.27"
 milestone:
-  alpha: "v1.24"
-  beta: "v1.25"
+  beta: "v1.24"
+  stable: "v1.26"
```

Comment:
It might be too late to raise but I am confused by the milestone labels here. Is this targeting beta in 1.27? If so, it seems not matching here. :)

Reply:
rollout of this KEP has three pieces, the first of which graduated to stable in 1.26 (LegacyServiceAccountTokenNoAutoGeneration)

LegacyServiceAccountTokenTracking graduates to beta in 1.27

the KEP template doesn't have a way to track progressive rollout of successive pieces

Reply:
Thanks for the clarification. In this case I guess the PRR section could be considered as approved earlier. Thanks